sage_session/encryptors/fernet_encrypt.py

```diff
@@ -0,0 +1,112 @@
+from .base import Encryptor
+
+try:
+    from cryptography.fernet import Fernet
+except ImportError:
+    raise ImportError("Install `cryptography` package. Run `pip install cryptography`.")
```

Comment:
Improve message here.

For example: to use `pkg name` encryption feature enable add cryptography ...